src/fn/index.ts

```diff
@@ -1,23 +1,24 @@
-export { dip } from "./dip"
-export { diode } from "./diode"
-export { cap } from "./cap"
-export { led } from "./led"
-export { res } from "./res"
-export { bga } from "./bga"
-export { soic } from "./soic"
-export { quad } from "./quad"
-export { qfn } from "./qfn"
-export { qfp } from "./qfp"
-export { mlp } from "./mlp"
-export { ssop } from "./ssop"
-export { tssop } from "./tssop"
-export { sot236 } from "./sot236"
-export { sot23 } from "./sot23"
-export { dfn } from "./dfn"
-export { pinrow } from "./pinrow"
-export { sot563 } from "./sot563"
-export { ms012 } from "./ms012"
-export { ms013 } from "./ms013"
-export { sot723 } from "./sot723"
-export { sod123 } from "./sod123"
-export { axial } from "./axial"
+export { dip } from "./dip";
+export { diode } from "./diode";
+export { cap } from "./cap";
+export { led } from "./led";
+export { res } from "./res";
+export { bga } from "./bga";
+export { soic } from "./soic";
+export { quad } from "./quad";
+export { qfn } from "./qfn";
+export { qfp } from "./qfp";
+export { mlp } from "./mlp";
+export { ssop } from "./ssop";
+export { tssop } from "./tssop";
+export { sot236 } from "./sot236";
+export { sot363 } from "./sot363";
+export { sot23 } from "./sot23";
+export { dfn } from "./dfn";
+export { pinrow } from "./pinrow";
+export { sot563 } from "./sot563";
+export { ms012 } from "./ms012";
+export { ms013 } from "./ms013";
+export { sot723 } from "./sot723";
+export { sod123 } from "./sod123";
+export { axial } from "./axial";
```

Comment:
hmm this file was reformatted with what looks like prettier, all tscircuit projects use biome though, can we reformat? I will add a github workflow to check for this in the near future

Reply:
yeah sure, do I undo the formating in this PR? or leave it as is ?

Reply:
Yep, I've removed the prettier formatting